GroupDocs.Editor.MVC/Products/Common/Util/Parser/YamlParser.cs

```diff
@@ -10,7 +10,8 @@ namespace GroupDocs.Editor.MVC.Products.Common.Util.Parser
 {
     public class YamlParser
     {
-        private static string YamlPath = Path.Combine(AppDomain.CurrentDomain.BaseDirectory, "configuration.yml");
+        private static string appRoot = AppContext.BaseDirectory.Substring(0, AppContext.BaseDirectory.LastIndexOf("bin"));
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Define the locale to be used in this string operation.](https://app.codacy.com/gh/groupdocs-editor/GroupDocs.Editor-for-.NET-MVC/pullRequest?prid=5172828)